### ISSUE TITLE
PMID:15218150 priority 

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#3688]
https://www.pombase.org/reference/PMID:15218150

**Comment by ValWood** [comment]
I'll hand this one over to you, it was on my to do list!

**Comment by ValWood** [comment]
(mating-type heterochromatin)

**Comment by PCarme** [comment]
This one is now waiting for approval. 
Some legacy annotations on Genetic Interactions need to be deleted, especially the ones stating interactions with deb1 (a confusion with rdp1, since it is also synonym name of deb1. But it actually is the RNA-directed RNA polymerase rdp1 that is studied here).

**Comment by ValWood** [comment]
reported to BioGRID.

I think, but I'm not sure, that if we make new, no specific interactions with mutants and phenotypes, the older redundant gene-gene interactions are automatically suppressed @kimrutherford. Is that correct?

Anyway I can check when this session is through (still need FYPO terms but soon)

**Comment by kimrutherford** [comment]
> I think, but I'm not sure, that if we make new, no specific interactions with mutants and phenotypes, the older redundant gene-gene interactions are automatically suppressed

I don't think that's what happens.  I can't see anything in the code to do that and I don't remember implementing it.

**Comment by ValWood** [comment]
OK in that case we should probably suppress the redundant annotations  in PomBase.

**Comment by kimrutherford** [comment]
Does redundant mean same genes and evidence code?

**Comment by ValWood** [comment]
yep. So some  are already annotated by BIogrid when we get to the session, but we make a new annotation with the  alleles and phenotypes.

**Comment by ValWood** [comment]
So a PRO term has:

[Term]
id: PR:000085722
name: histone H3.3, initiator methionine removed K36-trimethylated form (Schizosaccharomyces pombe)
def: "A histone H3.3, initiator methionine removed K36-methylated form (Schizosaccharomyces pombe) whose methylation type is trimethylation. Example: UniProtKB:P10651, 2-136, Lys-37, MOD:00083." [PRO:DAN, PomBase:VW, PMID:37459529]
comment: Category=organism-modification. Requested by=PomBase. Note that in the article cited residue number is different because the initial methionine is not counted.
synonym: "Spom-hht3/InitMet-/K36Me3" EXACT PRO-short-label [PRO:DAN]
synonym: "PRO_0000221367, Lys-37, MOD:00083" NARROW PRO-proteoform-ftid [PRO:DNx]
synonym: "UniProtKB:P10651, 2-136, Lys-37, MOD:00083" NARROW PRO-proteoform-std [PRO:DNx]
intersection_of: PR:000027604 ! histone H3.3, initiator methionine removed K36-methylated form (Schizosaccharomyces pombe)
intersection_of: has_part MOD:00083 ! N6,N6,N6-trimethyl-L-lysine
synonym: "hht3/Me3(K36)" EXACT PomBase_display_name [PomBase:curators]
property_value: pombase_gene_id SPBC1105.11c


we need 
synonym: "hht3/Me3(K36)" EXACT PomBase_display_name [PomBase:curators]
to be i nthe GO slot that is displayed

I guess if we have the ID we don't need to include any other labels? byt include "name:" if it is possible.

Doe that make sense?

**Comment by kimrutherford** [comment]
> So a PRO term has:

Was that comment for the other issue: pombase/pombase-chado#1173 ?

**Comment by ValWood** [comment]
yes!

**Comment by PCarme** [comment]
So, what should we do about that ? (Redundancy between some new interaction annotations made in Canto and some legacy annotations from BioGRID)

**Comment by kimrutherford** [comment]
> So, what should we do about that ? (Redundancy between some new interaction annotations made in Canto and some legacy annotations from BioGRID)

I will add an extra process to the nightly load that removes redundant interactions.  It's not implemented yet.  :-)
